permafrost/templates/permafrost/permafrostrole_form.html

```diff
@@ -103,8 +123,12 @@ <h6 class="m-0 text-primary font-weight-bold">
                 </div>
             </div>
             <div class="form-row border-bottom-gray-300">
-                <p class="col-6 font-weight-bold">{% trans 'Required Permissions' %}</p>
-                <p class="col-6 font-weight-bold">{% trans 'Optional Permissions' %}</p>
+                <p class="col-4 font-weight-bold">{% trans 'Required Permissions' %}</p>
```

Comment:
double-check bootstrap 5 font-weight utilities: https://getbootstrap.com/docs/5.0/utilities/text/#font-weight-and-italics

Reply:
fw-bold